contracts/upgrade/V3VoteScript.sol

```diff
@@ -201,24 +269,69 @@ contract V3VoteScript is OmnibusBase {
                 ))
             })
         });
+
+        votingVoteItems[9] = VoteItem({
+            description: "10. Transfer 508,106 MATIC 0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0 from Aragon Agent 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c to Liquidity Observation Lab (LOL) Multisig 0x87D93d9B2C672bf9c9642d853a8682546a5012B5",
+            call: ScriptCall({
+                to: params.finance,
+                data: abi.encodeCall(IFinance.newImmediatePayment, (
+                    params.maticToken,
+                    params.lolMultisig,
+                    params.maticAmountWeiForTransfer,
+                    params.transferReference
+                ))
+            })
+        });
     }
 
     function getVoteItems() public view override returns (VoteItem[] memory voteItems) {
         voteItems = new VoteItem[](VOTE_ITEMS_COUNT);
         uint256 index = 0;
 
+        // Time constraints checks using predeployed TimeConstraints contract
+        voteItems[index++] = VoteItem({
+            description: "1. Check DG voting enactment is after December 4, 2025 00:00:00 UTC",
```

Comment:
for all DG items, please put on ordinal number in the format "1.X. DESC-TEXT", where:
`1.` is for the DG-proposal 1
`X.` is the actual ordinal number of the item